modules/aws_ecs/variables.tf

```diff
@@ -15,11 +15,17 @@ variable "vpc_id" {
   description = "Select a VPC that allows instances access to the Internet."
 }
 
-variable "subnet_ids" {
+variable "private_subnet_ids" {
```

Comment:
deploying Retool on only public subnets is also, afaik, an explicitly supported deployment method  (https://docs.retool.com/self-hosted/quickstarts/ecs-fargate/cloudformation) -- can we make sure that is still supported after this change?

Reply:
You could, yes.  Just put the same two subnets in both lists.

Reply:
ah missed that the ingress tightening was only done for both RDS dbs, LGTM!